### PR TITLE
Exclude 3 more Q-ids from the default query

### PR DESCRIPTION
**Pull request by swedebugia**
These were present when I looked through swedish senses today

### CODE REVIEW

File: queries/default.sparql [hunk 1]
```diff
@@ -24,7 +24,17 @@ SELECT * WHERE {
   FILTER NOT EXISTS { ?item a wikibase:Property}.
   FILTER NOT EXISTS { ?item wdt:P131 ?_ }.
   FILTER NOT EXISTS {
-    VALUES ?not { wd:Q4167410 wd:Q101352 wd:Q11424 wd:Q3305213 wd:Q482994 wd:Q13417114 wd:Q7366 wd:Q208569 wd:Q15632617 wd:Q21191270 wd:Q5185279 }.
+    VALUES ?not { wd:Q4167410 wd:Q101352 wd:Q11424 wd:Q3305213 wd:Q482994 wd:Q13417114 wd:Q7366 wd:Q208569 wd:Q15632617 wd:Q21191270 wd:Q5185279 
+                # software
+                Q7397
```
Comment (Nudin) [inline queries/default.sparql:29]: The prefixes (`wd:`) are missing.

File: queries/default.sparql [hunk 2]
```diff
@@ -24,7 +24,17 @@ SELECT * WHERE {
   FILTER NOT EXISTS { ?item a wikibase:Property}.
   FILTER NOT EXISTS { ?item wdt:P131 ?_ }.
   FILTER NOT EXISTS {
-    VALUES ?not { wd:Q4167410 wd:Q101352 wd:Q11424 wd:Q3305213 wd:Q482994 wd:Q13417114 wd:Q7366 wd:Q208569 wd:Q15632617 wd:Q21191270 wd:Q5185279 }.
+    VALUES ?not { wd:Q4167410 wd:Q101352 wd:Q11424 wd:Q3305213 wd:Q482994 wd:Q13417114 wd:Q7366 wd:Q208569 wd:Q15632617 wd:Q21191270 wd:Q5185279 
+                # software
```
Comment (Nudin) [inline queries/default.sparql:28]: We might want to either add a comment to all items (preferred) or to none.

**Review by Nudin — CHANGES_REQUESTED** [review]
See my two inline comments.
But an even bigger issue is, that the query times out when adding these 5 additional rules. :/ Even without them the query sometimes timeouts. Interestingly it doesn't timeout (35s/50s) when the filter for the english language isn't included. We could remove that and filter the results ourself in python – but this sounds as if the query could be optimized. I'm not good at query-optimization though. Maybe @lucaswerkmeister can help?

**Comment by Nudin** [comment]
I managed to optimize the query enough to add a bigger list of false-positives I joined your list together with the list provided by @so9q to get rid of a lot of wrong matches!
Thanks for providing the list! It is merged and deployed now.

**Comment by lucaswerkmeister** [comment]
You could try changing `FILTER NOT EXISTS` to `MINUS`, that’s often faster and should be equivalent here.

**Pull request closed without merging**